android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -578,7 +579,7 @@ public long submitTransaction(@NonNull Transaction transaction)
             Util.logException(TAG, invalidTransactionException);
             throw invalidTransactionException;
         }
-        return txResponse.getBlockCount() - 1;
+        return blockCount;
```

Comment:
```suggestion
        return blockIndex;
```
For the same reason as above